pkg/cmd/opts/install.go

```diff
@@ -780,7 +780,35 @@ func (o *CommonOptions) InstallGcloud() error {
 	if runtime.GOOS != "darwin" || o.NoBrew {
 		return errors.New("please install missing gcloud sdk - see https://cloud.google.com/sdk/downloads#interactive")
 	}
-	return o.RunCommand("brew", "cask", "install", "google-cloud-sdk")
+	versionResolver, err := o.GetVersionResolver()
+	if err != nil {
+		return err
+	}
+	stableVersion, err := versionResolver.StableVersionNumber(versionstream.KindPackage, "gcloud")
+	if err != nil {
+		return err
+	}
+	clientURL := fmt.Sprintf("https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-%s-%s-%s.tar.gz", stableVersion, runtime.GOOS, runtime.GOARCH)
```

Comment:
This url returns a 404 error because I guess the architecture for `amd64` does not have a download link.
We have the https://cloud.google.com/sdk/docs/quickstart-linux which provides us with the following links to donwload the sdk but when we leave it to a customizable link that link does not exist. https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-281.0.0-linux-x86_64.tar.gz

Reply:
We will need to change the URL accordingly. For example you can map _amd64_ to _x86_64_